OtripleS.Web.Api.Tests.Unit/Services/TeacherAttachments/TeacherAttachmentServiceTests.Logic.cs

```diff
@@ -103,5 +102,49 @@ public async Task ShouldRetrieveTeacherAttachmentByIdAsync()
             this.loggingBrokerMock.VerifyNoOtherCalls();
             this.dateTimeBrokerMock.VerifyNoOtherCalls();
         }
+
+        [Fact]
+        public async Task ShouldRemoveTeacherAttachmentAsync()
+        {
+            // given
+            var randomTeacherId = Guid.NewGuid();
+            var randomAttachmentId = Guid.NewGuid();
+            Guid inputTeacherId = randomTeacherId;
+            Guid inputAttachmentId = randomAttachmentId;
+            DateTimeOffset inputDateTime = GetRandomDateTime();
+            TeacherAttachment randomTeacherAttachment = CreateRandomTeacherAttachment(inputDateTime);
```

Comment:
What's the point of having a datatime definition here?
For tests that don't need an explicit definition of date time, we should have a method that abstracts away that usage of date time - so I would call something like this:

```csharp
TeacherAttachment randomTeacherAttachment = CreateRandomTeacherAttachment();
```
This method has an overload where it becomes parameterless

Reply:
Do the same for the rest please